### PR TITLE
metrics: update the PD role expr in Overview Grafana

### PR DESCRIPTION
**Pull request by JmPotato**
Signed-off-by: JmPotato <ghzpotato@gmail.com>

### What problem does this PR solve?

tikv/pd#3449 changed the way to display the PD role in Grafana, which means the Overview pannel should also be updated.

### What is changed and how it works?

Use `pd_tso_role` to show the PD role.

### Related changes

tikv/pd#3449

### Check List 

Tests 

- Unit test
- Integration test
- Manual test

image

### Release note 

- Fix the PD role display error in Grafana Overview pannel.

### CODE REVIEW

File: metrics/grafana/overview.json [hunk 1]
```diff
@@ -5245,4 +5245,4 @@
   "title": "Test-Cluster-Overview",
   "uid": "eDbRZpnWk",
   "version": 1
-}
+}
```
Comment (Yisaer) [inline metrics/grafana/overview.json:5248]: add a new line

**Review by Yisaer — COMMENTED** [review]
Rest LGTM

File: metrics/grafana/overview.json (hunk 1 above)
Comment (JmPotato) [reply metrics/grafana/overview.json]: Done.

**Comment by JmPotato** [comment]
\cc @nolouch

**Review by Yisaer — APPROVED** [review]
LGTM

**Review by nolouch — APPROVED** [review]

**Pull request merged**